back-end/developerverse/models.py

```diff
@@ -1,3 +1,29 @@
+from django.conf import settings
 from django.db import models
 
 # Create your models here.
+
+class Project(models.Model):
+
+    # We cannot directly use 
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+    )
+    name = models.CharField(
+        max_length=140
+    )
+    repo_url = models.URLField(
+        max_length=480, blank=True
+    ) # allow for longer URLs
+    project_url = models.URLField(
+        max_length=480, 
+        blank=True
+    )
+    desc = models.TextField(
+        max_length=500, 
+        blank=True
+    ) # User can use about 80 words for description

```

Comment:
500 characters feels very short to me. 80 words is reasonable for an average or even above-average length, but that's tough as a hard limit. Let's do 3000 bytes.

Reply:
I changed it to 3000 characters which I believe should correspond to 3000 bytes.